### DISCUSSION TITLE
화면 줌인아웃시에 프로필 팝업 자동 호출 되는 현상

### DISCUSSION THREAD

**Discussion by mhkim0720** [naverz/zepeto-studio-kor#106] [5. Q&A]
월드에서 카메라 줌 인아웃 하다보면 프로필 팝업이 자동으로 계속 호출되는데 
이 현상을 막을 수 있는 방법이 있을까요? 
유니티 에디터에서는 발생하지 않는 현상이라 문의 드립니다. 

제페토 플레이어 카메라 설정에 벽 뚫는거 막기 위해 layer 를 everything으로 설정해놓은 상태입니다. 

답변 부탁드립니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 제페토 앱을 통해 월드를 play 도중에는
캐릭터에 대한 터치가 발생했을 때, 기본적으로 미니프로필 팝업이 호출됩니다.

그리고 이 때, 캐릭터 컨트롤러 모듈에서 ZepetoPlayers 컴포넌트의 MiniProfile 설정을 통해
이 기능을 on/off 할 수 있습니다.

(world 1.1.1 기준)
![miniprofile](https://user-images.githubusercontent.com/93751484/147041109-a8ea01b5-f414-4e92-8088-9466d61be5d3.png)
 
감사합니다.
